charts/rancher-aks-operator/rancher-aks-operator/100.0.0+up1.0.1/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+annotations:
```

Comment:
Since this is the final release we need to remove the rc asset from assets and from index.yaml